next-packages/styled-base/src/index.js

```diff
@@ -103,15 +103,31 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
       if (targetClassName !== undefined) {
         className += ` ${targetClassName}`
       }
+      let newProps = {}
 
-      const ele = React.createElement(
-        baseTag,
-        // $FlowFixMe
-        pickAssign(shouldForwardProp, {}, props, {
-          className,
-          ref: props.innerRef
-        })
-      )
+      for (let key in props) {
```

Comment:
was `pickAssign` much slower?

Reply:
It was a little slower, probably because it used `arguments`.